packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -105,6 +105,10 @@
       [dir=rtl] & {
         box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
       }
+
+      &::placeholder {
+        font-style: var(--terra-form-input-placeholder-font-style, italic);
```

Comment:
2 problems to fix with this one: 
- this is in the incorrect spot, need to move up and insert above the `&:focus` block at line 96, rather than being nested inside of it. 
- fix the theming variable names, add color, and remove the italic value for font-style:
```
&::placeholder {
      color: var(--terra-time-input-placeholder-color, #70787c);
      font-style: var(--terra-time-input-placeholder-font-style);
    }
```

Reply:
other quick update for line 83 -> change `color:` from `#000` to `#1c1f21`

Reply:
I had forgotten I left that in! Suggested changes implemented.

Reply:
Lets remove this line so color will correctly be imported from `terra-form-input` and will make `.time-input-hour, .time-input-minute` classes better match `.meridiem-display`:
- line 83 -> `     color: var(--terra-time-input-color, #000);`

Reply:
Need to update lines 50->55 to this for new updated values and add a manual override of the opacity (no need to make themeable) for the nested inputs which applies the opacity twice (if set by theming) when disabled.
```
    &.disabled {
      background-color: var(--terra-time-input-disabled-background-color, rgba(111, 116, 119, 0.2));
      border: var(--terra-time-input-disabled-border, 1px solid #868a8c);
      cursor: default;
      color: var(--terra-time-input-disabled-color, #1c1f21);
      opacity: var(--terra-time-input-disabled-opacity, 0.25);
      pointer-events: none;

      .time-input-hour[disabled],
      .time-input-minute[disabled],
      .meridiem-display[disabled]{
        opacity: 1;
      }
```